src/fhirtypes/ElementDefinition.ts

```diff
@@ -3076,6 +3090,15 @@ export type ElementDefinitionSlicingDiscriminator = {
 // Cannot constrain ElementDefinitionSlicingDiscriminator to have these values as a type
 // since we want to process other string values, but log an error
 const ALLOWED_DISCRIMINATOR_TYPES = ['value', 'exists', 'pattern', 'type', 'profile'];
+// R5 introduced the 'position' discriminator
+const ALLOWED_DISCRIMINATOR_TYPES_R5 = [
+  'value',
+  'exists',
+  'pattern',
+  'type',
+  'profile',
+  'position'
+];
```

Comment:
I don't have a strong opinion on this, but... it could be:
```suggestion
const ALLOWED_DISCRIMINATOR_TYPES_R5 = [...ALLOWED_DISCRIMINATOR_TYPES, 'position'];
```